src/js/poster-image.js

```diff
@@ -51,9 +51,13 @@ class PosterImage extends ClickableComponent {
 
       // Don't want poster to be tabbable.
       tabIndex: -1
-    }, {}, Dom.createEl('img', {
+    }, {
+      alt: 'some alt text'
+    }, Dom.createEl('img', {
       loading: 'lazy',
       crossOrigin: this.crossOrigin()
+    }, {
+      alt: ''
```

Comment:
This will insert `alt` rather than `alt=""`. Do we know if all validators would treat this as `null alt`?

Reply:
I don't think it will pass validation; I _think_ just inserting the attribute `alt` on an element sets alt to `true`, rather than an empty string, which isn't correct.

Would `alt: '""'` work?

Reply:
That ends up as `alt=""""` or `alt='""'` depending on the browser. On Chrome `el.setAttribute('alt', '')` will reset back to `alt` even if a string was previously set. Firefox has `alt=""`. I think as far as the browser is concerned `alt` and `alt=""` are equal. Lighthouse is happy with just `alt`.

Reply:
Even on Chome `el.getAttribute('alt')` returns `''` and `el.outerHTML` is `<img alt="">` so it ought to be ok... not sure if they're another way a validator could be testing that.

Reply:
With
```html
<!DOCTYPE html>
<html lang="en-US">
<head>
<title>foo</title>
<body>
<img alt src="https://example.com/foo.png">
</body>
</html>
```
the w3c validator passes https://validator.w3.org/nu/

I'd things to be fine as long as we're using `el.setAttribute('alt', '')`.

Reply:
And `Dom.createEl()` is using that.